pkg/executor/keptn.go

```diff
@@ -0,0 +1,96 @@
+package executor
+
+import (
+	"fmt"
+	"github.com/Azure/Orkestra/pkg/utils"
+	v1alpha13 "github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	KeptnImage = "azureorkestra/keptn-executor"
+	KeptnTag   = "v0.1.0"
+)
+
+type KeptnForward struct{}
+
+func (exec KeptnForward) Reverse() Executor {
+	return HelmReleaseReverse{}
+}
+
+func (exec KeptnForward) GetName() string {
+	return "keptn-forward-executor"
+}
+
+func (exec KeptnForward) GetTemplate() v1alpha13.Template {
+	return keptnBaseTemplate(exec.GetName(), Install)
+}
+
+func (exec KeptnForward) GetTask(name string, dependencies []string, timeout, hrStr *v1alpha13.AnyString) v1alpha13.DAGTask {
+	return keptnBaseTask(exec.GetName(), name, dependencies, timeout, hrStr)
+}
+
+type KeptnReverse struct{}
+
+func (exec KeptnReverse) Reverse() Executor {
+	return HelmReleaseForward{}
+}
+
+func (exec KeptnReverse) GetName() string {
+	return "keptn-reverse-executor"
+}
+
+func (exec KeptnReverse) GetTemplate() v1alpha13.Template {
+	return keptnBaseTemplate(exec.GetName(), Delete)
+}
+
+func (exec KeptnReverse) GetTask(name string, dependencies []string, timeout, hrStr *v1alpha13.AnyString) v1alpha13.DAGTask {
+	return keptnBaseTask(exec.GetName(), name, dependencies, timeout, hrStr)
+}
+
+func keptnBaseTemplate(executorName string, action Action) v1alpha13.Template {
```

Comment:
There is an additional input param that is required here - the reference to the configmap containing the Keptn configuration YAMLs. See this [code](https://github.com/nitishm/orkestra/blob/nitishm/keptn/executor/pkg/workflow/executor.go#L20)